modules/arkit/arkit_interface.mm

```diff
@@ -443,8 +465,18 @@
 				last_timestamp = current_frame.timestamp;
 
 				// get some info about our screen and orientation
-				Size2 screen_size = OS::get_singleton()->get_window_size();
-				UIInterfaceOrientation orientation = [[UIApplication sharedApplication] statusBarOrientation];
+				Size2 screen_size = DisplayServer::get_singleton()->screen_get_size();
+				UIInterfaceOrientation orientation = UIInterfaceOrientationUnknown;
+
+				if (@available(iOS 13, *)) {
+					orientation = [UIApplication sharedApplication].delegate.window.windowScene.interfaceOrientation;
+#if TARGET_OS_SIMULATOR
+				}
+#else
+				} else {
+					orientation = [[UIApplication sharedApplication] statusBarOrientation];
+				}
+#endif
```

Comment:
Another option:
```suggestion
#ifndef TARGET_OS_SIMULATOR
				} else {
					orientation = [[UIApplication sharedApplication] statusBarOrientation];
#endif
				}
```
But it might be a bit less readable :)